script/deployment/bash/deploy_main_contracts.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+source script/deployment/bash/common_functions.sh
+
+# Deploy BN256G2
+if [ "$USE_PRECOMPILED_MODEXP" = true ]; then
+    deploy_contract "src/common/BN256G2v1.sol:BN256G2v1"
+    BN256G2_ADDRESS=$(get_deployed_address "BN256G2v1")
+else
+    deploy_contract "src/common/BN256G2.sol:BN256G2"
+    BN256G2_ADDRESS=$(get_deployed_address "BN256G2")
+fi
+
+# Deploy BLS
+deploy_contract "src/common/BLS.sol:BLS"
+BLS_ADDRESS=$(get_deployed_address "BLS")
+
+# Deploy EOFeedVerifier
+FEED_VERIFIER_INIT=$(cast calldata "initialize(address,address,address,uint256,address[])" \
+    $OWNER_ADDRESS $BLS_ADDRESS $BN256G2_ADDRESS $EORACLE_CHAIN_ID "[$ALLOWED_SENDERS]")
+deploy_proxy "src/EOFeedVerifier.sol:EOFeedVerifier" $FEED_VERIFIER_INIT $PROXY_ADMIN_OWNER
+FEED_VERIFIER_PROXY=$(get_deployed_address "EOFeedVerifierProxy")
```

Comment:
```suggestion
FEED_VERIFIER_PROXY=$(get_deployed_address "feedVerifier")
```

Reply:
I agree that the naming is better with "Proxy", but we should be aligned with all previous deployments.
We can modify it at the end of the script, wdyt?

Reply:
Yes, I will change naming to be aligned with previous deployment. 
However I think naming format like `<contract_name>:<address>` and `<contract_name_Proxy>:<address>` is better. Not sure if need to change it since we have many deployments already